mobile/src/main.jsx

```diff
@@ -33,6 +35,8 @@ const store = createStore(
 )
 
 document.addEventListener('DOMContentLoaded', () => {
+  console.debug(__ALLOW_HTTP__)
```

Comment:
Sure? Obviously having a compil variable dedicated to it is probably a better practice than relying on the global `__TARGET__` one to do some feature-flip (see the @gregorylegarec point about it)